app/fissile.go

```diff
@@ -772,7 +773,7 @@ func compareHashes(v1Hash, v2Hash keyHash) *HashDiffs {
 
 // GenerateKube will create a set of configuration files suitable for deployment
 // on Kubernetes
-func (f *Fissile) GenerateKube(rolesManifestPath, outputDir, repository, registry, organization, fissileVersion string, defaultFiles []string, useMemoryLimits bool, opinions *model.Opinions) error {
+func (f *Fissile) GenerateKube(rolesManifestPath, outputDir, repository, registry, organization, fissileVersion string, defaultFiles []string, useMemoryLimits bool, createHelmChart bool, chartFilename string, opinions *model.Opinions) error {
```

Comment:
We are so far, so just avoid breaking any workflows that rely on them. They are mostly redundant though, except for the manual bosh-tasks, which aren't generated for the helm charts.

We have a card for the tasks (https://trello.com/c/hnfI8xtR/209-helm-charts-for-smoke-cats-etc); once that is done we can drop the old kube-config (after approval from all stakeholders).